build.gradle

```diff
@@ -10,7 +10,7 @@ buildscript {
             'composeNavigation': '1.0.0-alpha09',
             'detekt': '1.7.4',
             'espresso': '3.2.0',
-            'gradle': '4.2.0-alpha15',
+            'gradle': '7.0.0-alpha12',
```

Comment:
nit: to me at least it's confusing to name this `gradle` since it's not exactly the gradle version. the android gradle plugin (agp) is different from gradle